src/content/protocols/aave.md

```diff
@@ -0,0 +1,916 @@
+
+### Treasury, Aave Ecosystem Reserves and Rewards System
+
+This subsystem manages how fees are collected, spent and third-party rewards allocated and claimed by users. Fees taken across the Aave v3 protocol (and previous versions) are collected in the `TreasuryCollector`, a fully upgradeable contract. Through _Aave Governance_ and different (upgradeable) utility contracts these funds are distributed among service providers and ecosystem initiatives. Furthermore, third parties are able to distribute rewards, in their own ERC20 token, to Aave users through the `RewardsControllerManager` and `EmissionManager` contracts. Rewards are distributed continuously among eligible users but controlled by the _Aave Chan Initiative_ multisig account until claimed. The _Aave Chan Initiative_ multisig account can thus potentially be abused to withhold or steal distributed rewards resulting in a loss of unclaimed yield for users.
+
+### GHO Stablecoin
+
+`GHO` is Aave v3's native stablecoin that is pegged to the US Dollar and mintable at a 1:1 rate against the centralized `USDC` and `USDT` stablecoins through the `GhoStabilityModule` contract. Furthermore, a _Risk Council_ multisig account (not a _Security Council_ according to the DeFiScan definition) has the ability to mint unbacked `GHO` supply into (or burn from) the Aave lending market for users to borrow by supplying collateral assets (and thus collateralizing the `GHO` supply). Direct minting and burning through the _Risk Council_ is managed through the `GhoDirectMinter` contract and limited by minting caps which are controlled by _Aave Governance_. Both the `GhoStabilityModule` and the `GhoDirectMinter` as well as the respective `aToken` and `variableDebtToken` contracts upgradeable by _Aave Governance_ opening the possibility of uncontrolled minting or burning of `GHO` in case of an unintended upgrade. If abused, this control thus introduces a risk of loss of funds, loss of unclaimed yield (accrued interest in `GHO`) or an otherwise material impact on the expected protocol performance for `GHO` holders and Aave users.
```

Comment:
typo in "respective `aToken` and `variableDebtToken` contracts upgradeable", should be "respective `aToken` and `variableDebtToken` contracts are upgradeable"